src/System.Linq.Expressions/src/System/Dynamic/Utils/TypeExtensions.cs

```diff
@@ -19,15 +19,7 @@ internal static class TypeExtensions
         public static MethodInfo GetAnyStaticMethodValidated(this Type type, string name, Type[] types)
         {
             Debug.Assert(types != null);
-            foreach (MethodInfo method in type.GetMethods(BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Static | BindingFlags.DeclaredOnly))
-            {
-                if (method.Name == name && method.MatchesArgumentTypes(types))
-                {
-                    return method;
-                }
-            }
-
-            return null;
+            return type.GetMethod(name, BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Static | BindingFlags.DeclaredOnly, null, types, null);
```

Comment:
@VSadov, is this correct?  The corresponding code used in desktop appears to do subsequent validation of the argument types:
https://referencesource.microsoft.com/#System.Core/Microsoft/Scripting/Utils/TypeExtensions.cs,72